webapp/src/Controller/Team/ProblemController.php

```diff
@@ -134,6 +135,64 @@ public function problemTextAction(int $probId)
         return $response;
     }
 
+    /**
+     * @Route(
+     *     "/{probId<\d+>}/attachment/{attachmentId<\d+>}",
+     *     name="team_problem_attachment"
+     *     )
+     * @param int    $probId
+     * @param int    $attachmentId
+     * @return \Symfony\Component\HttpFoundation\Response
+     * @throws \Doctrine\ORM\NonUniqueResultException
+     */
+    public function attachmentAction(int $probId, int $attachmentId)
```

Comment:
Is this just duplicated code with above? Can you dedup it somewhere?

Reply:
I will try this

Reply:
I gave it a try and removed even more duplicated code. Can you check if you agree with the approach?